tcms/rpc/tests/test_bug.py

```diff
@@ -0,0 +1,51 @@
+import unittest
+from mock import patch, MagicMock
+
+from django.conf import settings
+
+from tcms.rpc.tests.utils import APITestCase
+
+if 'tcms.bugs.apps.AppConfig' not in settings.INSTALLED_APPS:
+    raise unittest.SkipTest('tcms.bugs is disabled')
+
+
+class TestBug(APITestCase):
+    def _fixture_setup(self):
+        super()._fixture_setup()
+        self.url = "http://some.url"
+        self.expected_result = {
+            'title': 'Bug from cache',
+            'description': 'This bug came from the Django cache'
+        }
+
+    @patch('tcms.rpc.api.bug.tracker_from_url')
+    @unittest.skip('debug')
```

Comment:
Don't skip this, looks like a valid scenario. Need to use cache.clean() before calling the API to guarantee no cached values will be used.

Reply:
yes, this is a leftover. the test is a valid one and should be executed